### PR TITLE
[DO-1055] Add to converter the without removing outdated case

### PR DESCRIPTION
**Pull request by Iva-khar**

### CODE REVIEW

File: data_ocean/converter.py [hunk 1]
```diff
@@ -246,7 +246,10 @@ def process(self, start_index=0):
         if records_len:
             self.save_to_db(records)
         if start_index == 0:
-            self.delete_outdated_companies()
+            try:
+                self.delete_outdated()
+            except:
+                print('This converter has not ability to remove outdated records')
```
Comment (roman-tiukh) [inline data_ocean/converter.py:252]: dont catch all exceptions

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: data_ocean/converter.py (hunk 1 above)
Comment (roman-tiukh) [reply data_ocean/converter.py:252]: you must understand what error you are catching

Comment (roman-tiukh) [reply data_ocean/converter.py:252]: also try hasattr or getattr builtins

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: data_ocean/converter.py (hunk 1 above)
Comment (Iva-khar) [reply data_ocean/converter.py]: refactored

**Pull request merged**